src/dotdom.js

```diff
@@ -37,7 +37,7 @@ module.exports = window;
    * @param {Array} [children] - The child VNode elements
    * @returns {VNode} Returns a virtual DOM instance
    */
-  let createElement = (element, props={}, ...children) => (
+  let c = (element, props={}, ...children) => (
```

Comment:
@kumavis yes, I'm aware of the scope requirement. However, I expect webpack/browserify and others to scope the modules for me. This module is a different design altogether as it's build target is IoT devices. And what I meant was.. the minifier should be able to scope the variables as if it were in scope.. Sorry, I did not expand on that point. Laziness. I understand it's top level scope and thus it wont change the var names.